fdbclient/ServerKnobs.cpp

```diff
@@ -1060,6 +1060,7 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( BLOB_RESTORE_MANIFEST_URL, isSimulated ? "file://simfdb/fdbblob/manifest" : "" );
 	init( BLOB_RESTORE_MANIFEST_FILE_MAX_SIZE, isSimulated ? 10000 : 10000000 );
 	init( BLOB_RESTORE_MANIFEST_RETENTION_MAX,                     10 );
```

Comment:
This is a bit outside the scope of this PR, but if we can restore to any point in time in the mutation log duration, but only had manifests for the most recent 10 manifest saves, how does a restore to a point older than the oldest manifest work if we wanted to restore to a previous point in time?
We should also buggify this knob down to 1 or 2 in simulation to ensure the manifest retention logic gets tested though.

Reply:
set a shorter retention for simulation env. verified that log truncation covered by correctness tests. I'll need to buggify a set of blob restore knobs in a separated JIRA. 

Regarding to point-in-time restore, the manifest includes \xff\x02/bgh system keys for time-travel reading of blob granules. so even if manifest is created at 2pm, we still can restore to 1pm(and ever earlier according to  purge policy) from readBlobGranules API.

Reply:
Makes sense.
We should probably add a code probe to ensure that this logic continues to be hit in future testing

Reply:
sounds good. can I add the code probe in next PR? :)

Reply:
sure

Reply:
In the same vein, we should add stats/counters to the blob manager for manifest flushes/manifest files flushed, granule flush+mutation log truncates, etc... to have some observability into this

Reply:
sure thing. will do it in next pr